sdk/communication/communication-chat/review/communication-chat.api.md

```diff
@@ -142,59 +185,77 @@ export interface ListPageSettings {
 }
 
 // @public
-export type ListReadReceiptsOptions = OperationOptions;
+export type ListParticipantsOptions = RestListParticipantsOptions;
 
 // @public
-export interface OperationResponse {
-    _response: HttpResponse;
-}
+export type ListReadReceiptsOptions = RestListReadReceiptsOptions;
 
 // @public
-export interface ReadReceipt extends Omit<RestReadReceipt, "senderId"> {
-    readonly sender?: CommunicationUserIdentifier;
+export interface OperationResponse {
+    _response: HttpResponse;
 }
 
 // @public
-export type RemoveMemberOptions = OperationOptions;
+export type RemoveParticipantOptions = OperationOptions;
 
 // @public
-export interface RestAddMembersRequest {
-    members: RestChatThreadMember[];
+export interface RestAddChatParticipantsRequest {
+    participants: RestChatParticipant[];
 }
 
-// @public (undocumented)
+// @public
 export interface RestChatMessage {
-    content?: string;
-    readonly createdOn?: Date;
+    content?: RestChatMessageContent;
+    createdOn: Date;
     deletedOn?: Date;
     editedOn?: Date;
-    readonly id?: string;
-    priority?: ChatMessagePriority;
+    id: string;
     senderDisplayName?: string;
-    readonly senderId?: string;
-    type?: string;
-    readonly version?: string;
+    senderId?: string;
+    sequenceId: string;
+    type: ChatMessageType;
+    version: string;
 }
 
-// @public (undocumented)
-export interface RestChatThread {
-    readonly createdBy?: string;
-    readonly createdOn?: Date;
-    readonly id?: string;
-    members?: RestChatThreadMember[];
+// @public
+export interface RestChatMessageContent {
+    initiator?: string;
+    message?: string;
+    participants?: RestChatParticipant[];
     topic?: string;
 }
 
 // @public
-export interface RestChatThreadMember {
+export interface RestChatMessageReadReceipt {
+    chatMessageId: string;
+    readOn: Date;
+    senderId: string;
+}
+
+// @public
+export interface RestChatParticipant {
     displayName?: string;
     id: string;
     shareHistoryTime?: Date;
 }
 
+// @public
+export interface RestChatThread {
+    createdBy: string;
+    createdOn: Date;
+    deletedOn?: Date;
+    id: string;
+    topic: string;
+}
+
+// @public
+export interface RestCreateChatThreadOptions extends coreHttp.OperationOptions {
+    repeatabilityRequestID?: string;
```

Comment:
@amrelroumy-msft  do we want to rename it in swagger?

Reply:
We are currently following this format from the Oasis spec https://docs.oasis-open.org/odata/repeatable-requests/v1.0/cs01/repeatable-requests-v1.0-cs01.html#sec_RepeatabilityRequestID.

It can be updated to be consistent with other occurrences of `Id`.